cmd/kubelet/app/options/options.go

```diff
@@ -331,6 +337,8 @@ func (f *KubeletFlags) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringVar(&f.NodeIP, "node-ip", f.NodeIP, "IP address of the node. If set, kubelet will use this IP address for the node")
 
+	fs.BoolVar(&f.ValidateNodeIP, "validate-node-ip", f.ValidateNodeIP, "Whether to validate if the node ip a valid ip address and assigned to the server")
```

Comment:
s/ip/IP/g

comment "Whether to validate that the node IP is assigned to an interface; disable this if the node is behind a NAT"